src/hearings/containers/request-hearing/hearing-timing/hearing-timing.component.spec.ts

```diff
@@ -722,7 +722,7 @@ describe('HearingTimingComponent', () => {
 
     component.setDataItems();
 
-    expect(component.duration).toBe(120);
+    expect(component.duration).toBe(60);
```

Comment:
Sorry not buying this test. On line 702 it says
component.sourceOfData = SourceOfData.SERVICE_HEARING_VALUES;
So the data should be coming from SHV, which has 120